webapp/channels/src/components/admin_console/system_user_detail/team_list/team_list_dropdown.tsx

```diff
@@ -52,18 +52,18 @@ const TeamListDropdown = ({
                     <Menu.ItemAction
                         id='makeTeamAdmin'
                         show={showMakeTeamAdmin}
-                        onClick={() => doMakeUserTeamAdmin(team.id)}
+                        onClick={useCallback(() => doMakeUserTeamAdmin(team.id), [team.id, doMakeUserTeamAdmin])}
                         text={intl.formatMessage({id: 'team_members_dropdown.makeAdmin', defaultMessage: 'Make Team Admin'})}
                     />
                     <Menu.ItemAction
                         show={showMakeTeamMember}
-                        onClick={() => doMakeUserTeamMember(team.id)}
+                        onClick={useCallback(() => doMakeUserTeamMember(team.id), [team.id, doMakeUserTeamMember])}
                         text={intl.formatMessage({id: 'team_members_dropdown.makeMember', defaultMessage: 'Make Team Member'})}
                     />
                     <Menu.ItemAction
                         id='removeFromTeam'
                         show={true}
-                        onClick={() => doRemoveUserFromTeam(team.id)}
+                        onClick={useCallback(() => doRemoveUserFromTeam(team.id), [team.id, doRemoveUserFromTeam])}
```

Comment:
Can we move this out? As in...
```
const TeamListDropdown({...}: Props) => {
    ...
    const makeTeamAdminOnClick = useCallback(() => ...);
    const makeTeamMemberOnClick = useCallback(() => ...);

    return (
        ...
            <Menu.ItemAction
                ...
                onClick={makeTeamAdminOnClick}
            />
       ...
    );
};
```

It is a bit more verbose, but helps not breaking the rule of hooks in the future.